pkg/gomod/modules.go

```diff
@@ -84,3 +87,17 @@ func Module(gomod string, domain string) (string, []string, error) {
 
 	return file.Module.Mod.Path, packages.List(), nil
 }
+
+func moduleToRepo(module string) (*git.Repo, error) {
+	url := fmt.Sprintf("https://%s?go-get=1", module)
+	meta, err := golang.GetMetaImport(url)
+	if err != nil {
+		return nil, fmt.Errorf("unable to fetch go import %s: %v", url, err)
+	}
+
+	if meta.VCS != "git" {
+		return nil, fmt.Errorf("unknown VCS: %s", meta.VCS)
```

Comment:
```suggestion
		return nil, errros.New("unknown VCS: "+meta.VCS)
```

Reply:
I don't understand why you like these changes so much. I think this is worse to read and implement

Reply:
less state. straight left to right )